apps/voucherify/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "voucherify-contentful-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "cross-env BROWSER=none react-scripts start",
+    "postinstall": "npm run build",
+    "build": "react-scripts build",
+    "install-ci": "npm ci",
+    "create-app-definition": "contentful-app-scripts create-app-definition",
+    "upload": "contentful-app-scripts upload --bundle-dir ./build",
```

Comment:
You could remove the `upload` and `upload-ci` scripts as they are redundant with the `deploy` scripts.